client/app/components/packages/attachments.js

```diff
@@ -9,18 +8,10 @@ import { action } from '@ember/object';
   * @param      {Package Model}  package
   */
 export default class PackagesAttachmentsComponent extends Component {
-  constructor() {
-    super(...arguments);
-
-    this.fileManager = this.fileManagement.findOrCreate(
-      this.args.package.id,
-      this.args.package.documents,
-    );
+  get fileManager() { // should be an instance of FileManager
+    return this.args.fileManager;
```

Comment:
Future REDO: We could potentially get fileManager from this.args.package.fileManager

Reply:
My thinking on this is that this is a move towards having the component class less aware of the internal structure of the package model.

If we have components that know a lot about specific methods of model, specific shapes, etc, they're a little less encapsulated, harder to test, and more prone to breakage should anything change.

Thoughts?